server/src/com/cloud/network/NetworkServiceImpl.java

```diff
@@ -3085,9 +3085,10 @@ public GuestVlan dedicateGuestVlanRange(DedicateGuestVlanRangeCmd cmd) {
         // Verify account is valid
         Account vlanOwner = null;
         if (projectId != null) {
-            if (accountName != null) {
-                throw new InvalidParameterValueException("accountName and projectId are mutually exclusive");
-            }
+            //accountName and projectId are mutually exclusive
```

Comment:
@nitin-maharana @ustcweizhou The scope field in UI should be fine. But since there is a breaking change (accountName is made optional from required), there may be scenarios which will break. As long as it is documented should be ok.